docs/using_the_api.md

```diff
@@ -171,7 +171,27 @@ query {
 }
 ```
 
-And that's how filters work :tada:
+You can also limit the number of results returned by your query using the `limit` flag. For example, lets say you wanted to fetch the first 30 dwellings of the
+last query:
+
+```
+query {
+  dwellings(limit:30 filters:[{field:dwellingForwardSortationArea comparator:eq value:"V5V"}, {field:evaluationHouseType comparator:eq value:"Single detached"}]) {
+    results {
+      forwardSortationArea
+      evaluations {
+        eghRating {
+          measurement
+          upgrade
+        }
+      }
+    }
+  }
+}
+```
+
+**Important note:** The limit applies to the dwellings returned, not evaluations. Since a dwelling can have multiple evaluations, you will likely receive more
+than 30. The default limit set by the pagination is 50 dwellings.
```

Comment:
We should also mention that regardless of what you say, there's a max limit of 300

Reply:
👍 good catch, just fixed that